packages/frontend/app/register/weekly-meetings/components/location-search.tsx

```diff
@@ -57,20 +57,63 @@ export function LocationSearch({
     await dispatch(getLocationSearchResultsThunk(form.getValues().query));
   };
 
+  const renderLocationCards = () => {
+    if (!state.response) {
+      return null;
+    }
+
+    return state.response.results.map(
+      (location: LocationSearchResult, i: number) => (
+        <button
```

Comment:
Could probably wrap this inside a scroll area so that you don't have to scroll all the way to the bottom to click on "use current location"

Reply:
It was already in a scroll area but height wasn't set. Good call.